README.md

```diff
@@ -74,7 +74,7 @@ see if some other plugin is using that mapping.
     * To use termite:<br/>
       `let g:gtfo#terminals = { 'unix' : 'termite -d' }`
     * To use rxvt-unicode:<br/>
```

Comment:
Ah, no, that's the lengthy name it officially goes by. However, its binary file goes by urxvt for convenience.